### PR TITLE
Adding filebeat on Mac OS X install support

### PR DESCRIPTION
**Pull request by Sauraus**

### CODE REVIEW

**Comment by Sauraus** [comment]
@cthayer actually `brew` is the default package manager for OSX in Chef, per https://docs.chef.io/resource_homebrew_package.html.

File: metadata.rb [hunk 1]
```diff
@@ -4,16 +4,17 @@
 license 'Apache-2.0'
 description 'Installs/Configures Elastic Filebeat'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '1.1.0'
+version '2.0.0'
```
Comment (cthayer) [inline metadata.rb]: This is a breaking change?  Not just a new feature addition?

I was expecting 1.2.0 ;)

Comment (Sauraus) [reply metadata.rb]: It adds new dependencies to the cookbook on `homebrew` which may be considered a breaking change?

Comment (cthayer) [reply metadata.rb]: Whether or not it's breaking is your call :)

I'd accept that argument if you are concerned the new dependencies might impact the pre-existing dependencies or recipes.

Comment (mitch-roblox) [reply metadata.rb]: We actually have Chef conventions on this. Updating dependencies is usually only a minor version bump: https://confluence.roblox.com/display/OP/Chef+Conventions#ChefConventions-Versioning

**Comment by Sauraus** [comment]
@cthayer I am adding the `CHANGELOG.md` entry but I am going to say that the up-stream cookbook owner does not want this done in the PR.

**Comment by mitch-roblox** [comment]
What I usually do is create a separate branch with just my changes (no version bump, no changelog) and do the upstream PR off that.

**Pull request merged**